### PR TITLE
feat(`@vtmn/svelte`): add focus event support for Svelte version of `VtmnTextInput`

### PR DESCRIPTION
**Pull request by BasileRaiwet**
## Changes description
Adding support of `focus` event on the `VtmnTextInput` for svelte.
This change has been archived by adding `on:focus` directive to the `input` and `textarea` elements.
## Context
We are currently using this component in Belgium (NFS) and we need to handle the focus event.
Actually we need to do it by manipulating the DOM and adding event listener manually, which is not great In terms of readability and maintainability.
## Checklist

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request directly from your main!
- [x] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
- [x] Check your code additions will fail neither code linting checks.
- [x] I have reviewed the submitted code.
- [x] I have tested on related showcases.

## Does this introduce a breaking change?
- No

### CODE REVIEW

File: packages/sources/svelte/src/components/forms/VtmnTextInput/test/VtmnTextInput.spec.js [hunk 1]
```diff
@@ -46,6 +46,17 @@ describe('VtmnTextInput', () => {
       expect(onChange.mock.results[0].value).toEqual(nextValue);
     });
 
+    test('Should trigger focus event if focused', async () => {
```
Comment (Tlahey) [inline packages/sources/svelte/src/components/forms/VtmnTextInput/test/VtmnTextInput.spec.js:49]: Thanks for your PR :) 
You can remove the async ^^

File: packages/sources/svelte/src/components/forms/VtmnTextInput/test/VtmnTextInput.spec.js [hunk 2]
```diff
@@ -208,6 +219,18 @@ describe('VtmnTextInput', () => {
       });
       expect(getIcon(container, 'unit-test')).toBeUndefined();
     });
+
+    test('Should trigger focus event if focused', async () => {
```
Comment (Tlahey) [inline packages/sources/svelte/src/components/forms/VtmnTextInput/test/VtmnTextInput.spec.js:223]: same :)

File: packages/sources/svelte/src/components/forms/VtmnTextInput/test/VtmnTextInput.spec.js (hunk 1 above)
Comment (BasileRaiwet) [reply packages/sources/svelte/src/components/forms/VtmnTextInput/test/VtmnTextInput.spec.js]: Done ! :)

**Review by Tlahey — APPROVED** [review]

**Comment by BasileRaiwet** [comment]
@Tlahey @thibault-mahe When can we expect this PR to be merged / deployed ? Just to get an idea, because it could really help us in a current development

**Review by lauthieb — APPROVED** [review]

**Comment by lauthieb** [comment]
Hi @BasileRaiwet, thanks for this PR! I just came back from vacation. For me that's ok, I approved. I just wait for the CI and when all it's ok, I merge it :)

**Pull request merged**